tests/console/gdb.pm

```diff
@@ -92,7 +97,6 @@ sub run {
     wait_serial_or_die("Quit anyway?");
```

Comment:
Perhaps the `Quit anyway?` message does not in some scenarios appear I would this change:
```suggestion
    wait_serial("Quit anyway?");
    type_string("y\n");
```

Reply:
It appears on all scenarios but I do thing your approach is better.

Reply:
The thing is it does appear in all scenarios. Even in GCE. I can add an extra 'y', but I think is papering over stuff. Pushed.